go.mod

```diff
@@ -56,6 +56,7 @@ require (
 	github.com/go-openapi/jsonreference v0.20.2 // indirect
 	github.com/go-openapi/swag v0.22.3 // indirect
 	github.com/go-task/slim-sprig/v3 v3.0.0 // indirect
+	github.com/gofrs/uuid v4.4.0+incompatible
```

Comment:
Can we please use this package here for UUID validation: https://github.com/google/uuid